airflow/ui/src/views/Docs.tsx

```diff
@@ -45,6 +45,7 @@ const Docs: React.FC = () => {
     { path: 'apache-druid', name: 'Apache Druid' },
     { path: 'apache-hdfs', name: 'Apache HDFS' },
     { path: 'apache-hive', name: 'Apache Hive' },
+    { path: 'apache-kafka', name: 'Apache Kafka' },
```

Comment:
I don't remember we ever edited a UI file when adding a provider?
cc @bbovenzi

Reply:
Yeah I would ignore this what's in `/ui` for now. It needs a refresh post 2.3